package/MDAnalysis/analysis/rms.py

```diff
@@ -335,43 +343,53 @@ def __init__(self, traj, reference=None, select='all',
         .. versionadded:: 0.7.7
         .. versionchanged:: 0.8
            *groupselections* added
+        .. versionchanged: 0.15.1
+            Refactor to fit with AnalysisBase API
         """
-        self.universe = traj
-        if reference is None:
-            self.reference = self.universe
-        else:
-            self.reference = reference
-        self.select = process_selection(select)
-        if groupselections is not None:
-            self.groupselections = [process_selection(s) for s in groupselections]
-        else:
-            self.groupselections = []
+        super(RMSD, self).__init__(atomgroup.universe.trajectory,
+                                   **kwargs)
+        self.universe = atomgroup
```

Comment:
So this *might* be a Universe, but it's safer to call `atomgroup.universe`.  You only call `select_atoms` off it later in the code, but it might be a headache one day if an attribute called universe was an AtomGroup :D